packages/block-library/src/cover/edit.js

```diff
@@ -469,6 +474,7 @@ function CoverEdit( {
 		<>
 			{ controls }
 			<Block.div className={ classes } data-url={ url } style={ style }>
+				<BoxControlVisualizer values={ styleAttribute?.padding } />
```

Comment:
If I understand properly, this adds some "visualization" when we're editing the values right?

@ellatrix This is the use-case I was talking about for adding "children" to wrappers.

I wonder how we could automatically add this in the padding "hook" without requiring the block to change its code.

Reply:
@youknowriad Indeed! This is for visualization 👍